oneclick_monolith_install/README.md

```diff
@@ -0,0 +1,5 @@
+## About One Click Monolith Install
```

Comment:
What if we just call this the "Monolith Install", and the manual installation becomes the secondary way you would do something more custom.

Reply:
Yeah, re. the name, we should definitely use another name. It's also not a 'wizard'. I was originally [using](https://commcare-cloud.readthedocs.io/en/latest/installation/index.html) `Quick Install`. I will update this.

Currently I have positioned this as below (per https://commcare-cloud.readthedocs.io/en/latest/installation/index.html)
> Most of the install is done automatically with configuration set to sensible defaults. This is useful to quickly get started, test or preview what’s involved in a working CommCareHQ environment.

The main reason why I am not comfortable positioning this as a production install is because it's not tested (other than on my local setup). The other less important reason is that the developer using this quick install would have no understanding of what happened in the install (which seems a bit necessary for production). 

(will be good to talk about this involving Mary too)